SPONSORS.md

```diff
@@ -15,3 +15,32 @@ The following organisations and individuals supported the [Wagtail's First Hatch
 - [The Motley Fool](https://www.fool.com/)
 - [R Strother Scott](https://twitter.com/rstrotherscott)
 - [Beyond Media](https://www.beyond.works/)
+
+The features below were developed thanks to the sponsorship of these organsations:
+
+ - Collections for images and documents - [University of South Wales](https://www.southwales.ac.uk/)
+
+- Display of revision differences - [Blackstone Chambers](https://www.blackstonechambers.com/)
+
+- Privacy settings on collections of documents - [Wharton Research Data Services](https://www.WhartonWRDS.com/) of [The Wharton School](https://www.wharton.upenn.edu)
+
+- Hooks for overriding menu actions - [The Motley Fool](https://www.fool.com/)
+
+- Improved page locking - [The Motley Fool](https://www.fool.com/)
+
+- Exporting data from reports, form submissions and ModelAdmin  - [The Motley Fool](https://www.fool.com/)
+
+- Moderation workflow  - [The Motley Fool](https://www.fool.com/)
+
+- Complex search queries  - [The Motley Fool](https://www.fool.com/)
+
+- Multi-lingual content - [Torchbox](https://torchbox.com/wagtail-cms/) and [The Mozilla Foundation](https://foundation.mozilla.org/en/)
+
+- Page aliases - [The Mozilla Foundation](https://foundation.mozilla.org/en/)
+
+- Hooks for Snippet customization - [The Mozilla Foundation](https://foundation.mozilla.org/en/)
+
+- Utility for testing hooks - [The Mozilla Foundation](https://foundation.mozilla.org/en/)
+
+- Telepath for StreamField editing - [YouGov](https://yougov.co.uk/)
```

Comment:
I think this is missing commenting from 2.13?

Reply:
Oops you are right. Added it